packages/origin-ui-core/src/components/Organization/IRECConnectOrRegisterStep.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Paper, Grid, Box, Button, Theme, useTheme } from '@material-ui/core';
+import { STEP_NAMES } from './OrganizationRegistrationStepper';
+import OrgAddedIcon from '../../../assets/icon-org-added.svg';
+import { useTranslation } from '../..';
+import { Trans } from 'react-i18next';
+
+interface IProps {
+    nextStep: (step: STEP_NAMES) => void;
+}
+
+export const IRECConnectOrRegisterStep = (props: IProps) => {
+    const { nextStep } = props;
+    const { t } = useTranslation();
+    const { spacing }: Theme = useTheme();
+
+    return (
+        <Paper elevation={1} style={{ padding: spacing(2) }}>
+            <Grid container direction="column">
+                <Grid item container>
+                    <Grid item xs={3}>
+                        <img src={OrgAddedIcon} style={{ maxWidth: '100%', maxHeight: '100%' }} />
+                    </Grid>
+                    <Grid item xs={9}>
+                        <Box fontWeight="fontWeightBold">
+                            {t('organization.registration.titleConnectOrRegisterIREC')}
+                        </Box>
+                        <br />
+                        <Box style={{ fontWeight: 'lighter' }}>
+                            <Trans i18nKey="organization.registration.contentConnectOrRegisterIREC">
+                                We are checking your information as soon as possible and will
+                                contact you once everything is approved and you can start trading.
+                                <br /> <br /> In order to register devcices and request I-RECs,
+                                users also need to connect to existing I-REC account
```

Comment:
Can't we default to English instead of defining this string twice?

Reply:
Not sure how to avoid this duplication. In the documentation there are two ways of providing default values - by using children or default property. Could you point out how this can be done?

Reply:
I thought it already automatically defaults to English if you use `t('organization.registration.contentConnectOrRegisterIREC')`. Could you check if it does or doesn't?

Also,

Reply:
The problem is that I also use this child element to interpolate the html nodes. The documentation mentions that it can also be done through `components` property, but when I try to list nodes like `components = {[', <br />, <br />]}` eslint complains that the `Missing" key "prop for element in array`. May be `components` property should be used in some other way, but the documentation on this topic is very sparse.

Reply:
If I understand it correctly, the problem is that we're using HTML elements in the translation e.g. `<br />`?

HTML shouldn't be a part of the translation JSON files.

Why not split the translation into 2 different translation strings, and then do `<br />` only in HTML? (or better yet, use margins and paddings instead of `<br />`.

Reply:
The initial source of confusion is the need to create multiple keys in the JSON file if the text contains HTML nodes. react i18next solves this problem using the Trans component and for this I need to list the node indices in the JSON file

Reply:
Do you think that it is better to have multiple keys for logically one text than to duplicate translation?

Reply:
Not sure. For e.g. now we have a typo that needs to be fixed in 2 different places because the strings are duplicated.
We can leave it as-is, just make sure to fix the typo in both places.

If there's no good solution here, let's then leave this be. 👍

Reply:
Yes, the optimal solution would be to have translation string only in JSON and in Trans only list nodes but I can't find way to do this